server/services/utils/getServicesForUser.ts

```diff
@@ -194,13 +200,7 @@ export default (
       heading: 'Adjudications',
       description: 'Place a prisoner on report after an incident, view reports and manage adjudications.',
       href: config.serviceUrls.manageAdjudications.url,
-      enabled: () => {
-        const activeEstablishment = isActiveInEstablishment(activeCaseLoadId, ServiceName.ADJUDICATION, activeServices)
-        if (activeEstablishment === undefined)
-          return config.serviceUrls.manageAdjudications.enabledPrisons.split(',').includes(activeCaseLoadId)
-
-        return activeEstablishment
-      },
+      enabled: () => isActivitiesEnabled,
     },
```

Comment:
^ Should this be Adjudications, not Activities??

Reply:
Bound to get wires crossed with all of Activities, Appointments and Adjudications starting with 'A'!

Reply:
Good spot!